src/app/progress-bars/progress-bar-inline.ts

```diff
@@ -43,7 +42,7 @@ export class ProgressBarInlineDemo implements OnInit {
 
     runProgressBar(): void {
         this.stopProgressBar();
-        this.inlineProgressTimerId = setInterval(() => {
+        this.inlineProgressTimerId = window.setInterval(() => {
```

Comment:
Why add `window` here? Not that we'll ever compile this sucker on a server, but it would fail.

Reply:
Ah, it wasn't passing the linter (or type checker?) without it. Let me see if that's still the case and find ways to make it work without the `window`.